### PR TITLE
Raidboss: Swallow's Compass timeline and triggers

### PR DESCRIPTION
**Pull request by JLGarber**
Slightly late for BLU users, but here it is. All the special stuff is in the in-line comments.

### CODE REVIEW

File: ui/raidboss/data/04-sb/dungeon/swallows_compass.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+      response: Responses.stackMarkerOn('info'), // Info rather than alert to avoid collision with Both Ends.
+    },
+    {
+      // The Long end is a knockback in phase 1, but not in phase 2.
+      // Using the source name for tethers runs into localizing issues,
+      // so we just track the phase instead.
+      // The ability use here is unnamed, the teleport to the center to begin the intermission.
+      id: 'Swallows Compass Intermission Tracking',
+      netRegex: NetRegexes.ability({ id: '2CC7', source: 'Qitian Dasheng', capture: false }),
+      condition: (data) => !data.seenIntermission,
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/dungeon/swallows_compass.js:153]:
```suggestion
```

File: ui/raidboss/data/04-sb/dungeon/swallows_compass.txt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+2483.0 "Both Ends"
+2494.6 "Mount Huaguo"
+2508.9 "Equal Of Heaven"
+2511.0 "The Long End"
+2522.2 "The Short End"
+
+
+# Any time one shadow is reduced to 0% HP,
+# the other teleports to the center and casts Splitting Hairs.
+# Upon resolution, the shadows pick up approximately where they left off.
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/dungeon/swallows_compass.txt:133]: Foof! Thanks for handling this and all the comments.

**Comment by quisquous** [comment]
Do you have any time to do the oopsy file for this as well? Or, can you send me some logs and I can take a look?

**Comment by JLGarber** [comment]
I can probably do that tomorrow, yes.

**Pull request merged**